src/Constants.mac.cs.in

```diff
@@ -180,5 +180,9 @@ namespace ObjCRuntime {
 		public const string ScreenTimeLibrary = "/System/Library/Frameworks/ScreenTime.framework/ScreenTime";
 		public const string UniformTypeIdentifiersLibrary = "/System/Library/Frameworks/UniformTypeIdentifiers.framework/UniformTypeIdentifiers";
 		public const string UserNotificationsUILibrary = "/System/Library/Frameworks/UserNotificationsUI.framework/UserNotificationsUI";
+
+		// macOS 11.0
+		public const string AppTrackingTransparencyLibrary = "/System/Library/Frameworks/AppTrackingTransparency.framework/AppTrackingTransparency";
```

Comment:
This is really minor, but this line should go in the section above (the macOS 10.16/11.0 section, just after "AccessibilityLibrary", since we try to keep the entries sorted alphabetically).